xml/System.Web.Security.AntiXss/AntiXssEncoder.xml

```diff
@@ -180,11 +180,11 @@
   
 |Input examples| Encoded outputs|  
 |-|-|  
-|`alert('XSS Attack!');`|`alert(&#39;XSS&#32;Attack!&#39;);`|  
+|`alert('XSS Attack!');`|`alert(&#39;XSS Attack!&#39;);`|  
```

Comment:
I don't see the space character listed as a safe character in the table that starts on line 142. Does it need to be added?